### ISSUE TITLE
code splitting broken on Windows

### ISSUE THREAD

**Issue by igisev** [ezolenko/rollup-plugin-typescript2#169]
#### What happens and why it is wrong

Briefly:
Code splitting is not supported now

#### Versions
- typescript: 3.5.3
- rollup: 1.20.3
- rollup-plugin-typescript2: 0.24.0

#### rollup.config.js

```js
import resolve from 'rollup-plugin-node-resolve';
import ts2 from 'rollup-plugin-typescript2';

export default {
  // NOTE: object here
  input: {
    'my-lib': 'src/main.ts',
    'my-tools': 'src/tools.ts'
  },
  output: {
    dir: 'dist',
    format: 'esm'
  },
  plugins: [
    resolve(),
    ts2()
  ]
};
```

#### tsconfig.json

```json5
{
  "compilerOptions": {
    "target": "esnext",
    "module": "esnext",
    "strict": true,
    "sourceMap": true,
    "importHelpers": true,
    "downlevelIteration": true,
    "moduleResolution": "node",
    "experimentalDecorators": true,
    "allowSyntheticDefaultImports": true,
    "esModuleInterop": true,

    "lib": ["esnext"],
    "types": ["node"]
  },
  "include": ["src/**/*.ts"],
  "exclude": ["node_modules"]
}
```

#### files

src/tools.ts
```ts
export function myFn() {
    console.log('Hello World!');
}
```

src/main.ts
```ts
import { myFn } from './tools';

console.log(myFn());
```

#### plugin output

dist/my-tools.js **OK**
```js
function myFn() {
    console.log('Hello World!');
}

export { myFn };
```

dist/my-lib.js **WRONG**
```js
function myFn() {
    console.log('Hello World!');
}

console.log(myFn());
```

#### expected output

dist/my-lib.js
```js
import { myFn } from './my-tools.js';

console.log(myFn());
```

**Comment by ezolenko** [comment]
Could you try the same thing with plain rollup, is it supposed to work? (turn ts into js manually and remove rpt2 from plugins)

**Comment by igisev** [comment]
> Could you try the same thing with plain rollup, is it supposed to work?

This is a working well-tested feature in Rollup.
What is the point of checking it?
Rollup has a boilerplate: https://github.com/rollup/rollup-starter-code-splitting

Ok
case1:
- rollup-plugin-typescript2 is off
- all files renamed from ts to js

case2:
- no plugins at all
- all files renamed from ts to js

_Result: output for case1 and case2 is identical (see below)_

#### rollup.config.js

```javascript
import resolve from 'rollup-plugin-node-resolve';

export default {
  // NOTE: object here
  input: {
    'my-lib': 'src/main.js',
    'my-tools': 'src/tools.js'
  },
  output: {
    dir: 'dist',
    format: 'esm'
  },
  plugins: [
    resolve()
  ]
};
```

#### output

dist/my-tools.js
```javascript
function myFn() {
  console.log('Hello World!');
}

export { myFn };
```

dist/my-lib.js
```javascript
import { myFn } from './my-tools.js';

console.log(myFn());
```

**Comment by ezolenko** [comment]
I see the problem, rollup doesn't normalize paths when using them as keys for code splitting. Workaround is to add `rollupCommonJSResolveHack: true` to rpt2 options (solves a similar problem with another plugin, they fixed it in recent release actually).

I'll open rollup bug.

**Comment by igisev** [comment]
It seems rollup is not going to fix.
@ezolenko, please. Can you solve this problem yourself?
This feature is critical for us.

**Comment by ezolenko** [comment]
@igisev can you use `rollupCommonJSResolveHack: true` as a workaround?

**Comment by agilgur5** [comment]
So this should have been fixed by #251, which was released in [0.30.0](https://github.com/ezolenko/releases/0.30.0).

OP never responded regarding using `rollupCommonJSResolveHack` as a workaround, but #251 seems to state the same problem and basically makes the behavior of `rollupCommonJSResolveHack` always on.

So going to close this out as fixed.
